architecture/architecture-component-imageregistry.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * n/a
+
+[id='architecture-component-imageregistry']
```

Comment:
It looks like we can update the existing module(s) and include them here:

include::modules/registry-integrated-openshift-registry-registry-options.adoc[leveloffset=+1]

Do you want to mention any of the other registries (third party, Quay, registry.redhat.io)?

Reply:
not under product architecture no.